deli fixes

JeffBobbo opened the pull request:
Geometry fixes/improvements:
* Missing cubes under the ramps up to flag points
* Corrupt geometry in kaboom rocket rooms (might of been me, acerspyro noticed it when it was fixed on my version)
* Fencing above each base spawn room (room with mine and ramp)
* Replaced the open top of the rocket spawn shaft with geometry and sky texture (inhibit flag reset toss).
* Fixed some windows outside of the map
* Made the archways consistent.

Texture fixes:
* The ground level hallway of the flag building
* The fences above each team's spawn room
* Crate texture showing on wall adjacent to ladder on flag point level of each base
* Some instances of sky texture being visible from inside of the map
* Some inconsistencies with textures in lots of locations
* Added wooden boards/window texture on geometry that was indented for windows
* Fixes in the central room (rocket spawn shaft) and the corridors leading to them

Material fixes/improvements:
* Removed a lot of death material (to stop flag reset abuse).
  * Placed death material only on the outside of the map above ground (so flags reset when tossed outside of the map, but grenades don't explode when they go outside)
* Improved clip around the lampposts
* Reclipped a lot of the map boundaries (had to be done because of death removal)
* Improved clip in kaboom rocket rooms

shirepirate commented:
acute angled geometry gets clipped when possible, no exceptions. in some cases abuse is possible when this isn't done. so replace the clip in the dark corner near the top side hallway of alpha base, to the left of the flag when facing alpha base

you've unclipped a large area in the top of the center geometry, alpha side (a curved roof). this was clipped before and should probably stay that way, it only gives a large new spot to flag camp or hide and it's largely inaccessible except to pros.

also there's a section of clip that's too large above the omega spawn room, allowing you to stand in mid-air (this was there from before). fix

JeffBobbo commented:
Fixed points 2 and 3.
![alphacurve](https://cloud.githubusercontent.com/assets/5425481/7101509/b0a4729a-e054-11e4-870c-e3fd0c8cbd91.png)
![omegaspawnroom](https://cloud.githubusercontent.com/assets/5425481/7101516/d9a7ac48-e054-11e4-9867-5a72b3ee31de.png)

I'm not sure what you are referring to by point one, but I assume this (which I've now fixed).
![grenaderoom](https://cloud.githubusercontent.com/assets/5425481/7101513/cd842360-e054-11e4-94bb-3104e1e5c91f.png)

I noticed some lamps had noclip material, but weren't actually being clipped. I removed the material and set noclip flag on all the lamps.
![lamps](https://cloud.githubusercontent.com/assets/5425481/7101523/1bb0b45e-e055-11e4-8922-ec5ba30b1acc.png)

I also clipped these windows, and done some consistency fixes (the ones you're looking at had a bigger recess.
![windowclip](https://cloud.githubusercontent.com/assets/5425481/7101540/ff65682a-e055-11e4-8c51-a0897ca6d42d.png)

Since it's impossible to stand on these slops (under default variables), I've unclipped them (and noclipped the little bit of geometry under them, so you just slide off)
![alphaslope](https://cloud.githubusercontent.com/assets/5425481/7101558/9077fa4e-e056-11e4-8b8b-2aaf118a84c6.png)
and then used that to clean up the clip above the railings on alpha side
![alpharailing](https://cloud.githubusercontent.com/assets/5425481/7101561/9f1206a8-e056-11e4-821e-4820c3b1df5d.png)

shirepirate commented:
there's a dark corner next to the far wall which you redid the clip/noclip on, that used to have clip. side hallway on upper edge of the map
